keras_cv/models/backbones/backbone_presets.py

```diff
@@ -15,15 +15,18 @@
 
 from keras_cv.models.backbones.resnet_v1 import resnet_v1_backbone_presets
 from keras_cv.models.backbones.resnet_v2 import resnet_v2_backbone_presets
+from keras_cv.models.object_detection.yolo_v8 import yolo_v8_backbone_presets
 
 backbone_presets_no_weights = {
     **resnet_v1_backbone_presets.backbone_presets_no_weights,
     **resnet_v2_backbone_presets.backbone_presets_no_weights,
+    **yolo_v8_backbone_presets.backbone_presets_no_weights,
```

Comment:
Would these backbones work with RetinaNet?

Reply:
Yes they would, and all of the other backbones would work with YOLOV8.

That said, Luke mentioned that we should keep these YOLOV8 backbones out of the main preset dict, so that's fine by me.

Reply:
Good question @LukeWood. Originally my plan was for every Task model to maintain a separate dict of presets at the backbone level. However, it seemed like we could construct a dense graph of Tasks and Backbones that were mutually compatible---mainly traditional convolutional based architectures---and reuse them in several places. For this reason I am omitting ViT from `backbone_presets` but in my head I'm thinking of them as `traditional_backbone_presets`. Alternative suggestions welcome.